FlowCryptCommon/Extensions/CalendarExtensions.swift

```diff
@@ -9,8 +9,15 @@
 import Foundation
 
 public extension Calendar {
-    
     func isDateInCurrentYear(_ date: Date) -> Bool {
-        return self.isDate(date, equalTo: Date(), toGranularity: .year)
+        self.isDate(date, equalTo: Date(), toGranularity: .year)
+    }
+}
+
+public extension Bundle {
+    
+    static var flowCryptBundleTypeExtension: String {
+        guard let bundleIdentifier = Bundle.main.bundleIdentifier else { return "aaa" }
+        return bundleIdentifier ?? "N?A"
```

Comment:
It seems this property isn't used anywhere, so can be deleted, I think
(also `aaa` is a bit strange return value :)

Reply:
Merged master and replaced test code. Check latest commit